### PR TITLE
ci: improve go.build in Makefile, setup multi-arch docker image build…   

### PR DESCRIPTION
**Pull request by SignorMercurio**
…ing and pushing, improve release workflow.

Master issue: #1265

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -35,40 +41,59 @@ jobs:
...
+            libbtrfs-dev libgpgme-dev libdevmapper-dev \
+            libbtrfs-dev:arm64 libgpgme-dev:arm64 libdevmapper-dev:arm64
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v3
+        with:
+          args: release --rm-dist --release-notes=scripts/release/Note.md
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
```
Comment (cuisongliu) [inline .github/workflows/release.yml:80]: add makefile make oss ?

File: .goreleaser.yml [hunk 2]
```diff
@@ -133,6 +133,13 @@ builds:
       - -trimpath
       - -s -w
 
+release:
+  github:
+    owner: labring
+    repo: sealos
+  draft: true
```
Comment (cuisongliu) [inline .goreleaser.yml:140]: If set to true, will not auto-publish the release.

draft Is this setting inappropriate?

**Review by cuisongliu — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (SignorMercurio) [reply .github/workflows/release.yml:80]: I'd like to test it before add steps to push binaries to oss.

File: .goreleaser.yml (hunk 2 above)
Comment (SignorMercurio) [reply .goreleaser.yml:140]: Yes, for testing purposes. After the release workflow is stable, I'll change `draft` to `false`.

**Review by fanux — APPROVED** [review]

**Comment by SignorMercurio** [comment]
Ready to merge

**Review by zzjin — APPROVED** [review]
/LGTM

**Pull request merged**